hw/ip/otp_ctrl/doc/_index.md

```diff
@@ -8,14 +8,285 @@ title: "OTP Controller Technical Specification"
 This document specifies the functionality of the one time programmable (OTP) memory controller.
 The OTP controller is a module that is a peripheral on the chip interconnect bus, and thus follows the [Comportability Specification]({{< relref "doc/rm/comportability_specification" >}}).
 
+The OTP is a module that provides a device with one-time-programming functionality.
+The result of this programming is non-volatile, and unlike flash, cannot be reversed.
+The OTP functionality is constructed through an open source OTP controller and a proprietary OTP IP.
+
+The OTP controller provides:
+- An open-source abstraction interface that software can use to interact with a proprietary OTP block underneath.
+- An open-source abstraction interface that hardware components (for example life cycle controller and key manager (**TODO: add links here**)) can use to interact with a proprietary OTP block underneath.
+- High level logical security protection, such as integrity checks and scrambling of sensitive content.
+- Software isolation for when OTP contents are readable and programmable
+
+The proprietary OTP IP provides:
+- Reliable, non-volatile storage.
+- Inclusive of any technology-specific redundancy or error correction mechanisms.
```

Comment:
i think we can get rid of `inclusive of any`

Reply:
yeah let's remove that.